oauth-common/src/main/java/io/strimzi/kafka/oauth/jsonpath/ListNode.java

```diff
@@ -0,0 +1,30 @@
+/*
+ * Copyright 2017-2019, Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.kafka.oauth.jsonpath;
+
+import java.util.ArrayList;
+import java.util.stream.Collectors;
+
+class ListNode extends Node {
+    private final ArrayList<Node> items;
+
+    ListNode(ArrayList<Node> items) {
+        this.items = items;
+    }
+
+    @Override
+    public String toString() {
+        return items.stream().map(Object::toString).collect(Collectors.joining(",", "[", "]"));
```

Comment:
It's this going to produce the same result as simply calling `items.toString`?

Reply:
So true :D